src/orange/detail/UnitInserter.cc

```diff
@@ -303,7 +303,7 @@ UniverseId UnitInserter::operator()(UnitInput&& inp)
         // Create a set of background volume ids for BIH construction
         if (inp.volumes[i].flags & VolumeRecord::Flags::implicit_vol)
         {
-            background_vol_ids.insert(LocalVolumeId{i});
+            background_vol_ids.insert(LocalVolumeId(i));
```

Comment:
There's a newish helper class for constructing IDs from possibly different integer types:
```suggestion
            background_vol_ids.insert(id_cast<LocalVolumeId>(i));
```